grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -45,8 +46,12 @@ class DocumentTemplateService {
         Context context = new Context()
         context.putVar("invoiceItems", shipmentInstance?.shipmentItems)
         context.putVar("datePrinted", Constants.EUROPEAN_DATE_FORMATTER.format(new Date()))
```

Comment:
Why are we hard-coding the date formatter here? I feel like this should be locale specific.

Reply:
This is good question, I do not remember